website/docs/cdktf/create-and-deploy/configuration-file.mdx

```diff
@@ -206,3 +207,15 @@ This generates the `aws` provider bindings in the folder `./imports`. This is us
   "codeMakerOutput": "imports"
 }
 ```
+
+### Enable Crash Reporting for the CLI
+
+You can enable or disable crash reporting by setting the `sendCrashReports` property to `true` or `false`. Sending crash reports helps our team improve the CLI faster, you can learn more about what we track on our [telemetry page](/cdktf/telemetry#crash-reporting).
+
+```json
+{
+  "language": "typescript",
+  "app": "npm run --silent compile && node main.js",
+  "sendCrashReports": "true"
```

Comment:
This example might be a bit misleading, as the property is parsed as a JSON and treated as a boolean.
And if I would write `"sendCrashReports": "false"` it would still enable them.
We should either catch `"false"` (preferred) or change this example to `"sendCrashReports": true`.